Makefile

```diff
@@ -1,34 +1,6 @@
-SHELL := /usr/bin/env bash
-OPERATOR_DOCKERFILE = build/Dockerfile
+include boilerplate/generated-includes.mk
 
-# Include shared Makefiles
-include project.mk
-include standard.mk
```

Comment:
This commit should delete these files.

Reply:
Do you mean they will not be included in the `generated-includes.mk` file anymore.

Reply:
`generated-includes.mk` brings in the `project.mk` and `standard.mk` from the _convention_. These are obsolete. That's one of the main things we've actually got working right now :)

Reply:
I don't understand. Do you mean that the files in the `boilerplate` directory should be deleted? Because I still see them in the boilerplate repo.

Reply:
No. The ones in the repo root. The last two here:

```
[efried@efried deadmanssnitch-operator]$ git ls-files | grep mk$
boilerplate/_lib/boilerplate.mk
boilerplate/generated-includes.mk
boilerplate/openshift/golang-osd-operator/project.mk
boilerplate/openshift/golang-osd-operator/standard.mk
project.mk
standard.mk
```

Reply:
Got it. Thanks. Removed.